local run test failed with traefik helm install error

```sh
$ make create-all
====== create-k3d-cluster ======
INFO[0000] portmapping '15080:80' targets the loadbalancer: defaulting to [servers:*:proxy agents:*:proxy] 
INFO[0000] portmapping '15443:443' targets the loadbalancer: defaulting to [servers:*:proxy agents:*:proxy] 
INFO[0000] portmapping '15000:80' targets the loadbalancer: defaulting to [servers:*:proxy agents:*:proxy] 
INFO[0000] Prep: Network                                
INFO[0000] Created network 'k3d-cosmo'                  
INFO[0000] Created image volume k3d-cosmo-images        
INFO[0000] Creating node 'cosmo.io'                     
INFO[0000] Successfully created registry 'cosmo.io'     
INFO[0000] Starting new tools node...                   
INFO[0000] Starting Node 'k3d-cosmo-tools'              
INFO[0001] Creating node 'k3d-cosmo-server-0'           
INFO[0005] Pulling image 'rancher/k3s:v1.24.11-k3s1'    
INFO[0012] Creating node 'k3d-cosmo-agent-0'            
INFO[0012] Creating node 'k3d-cosmo-agent-1'            
INFO[0012] Creating LoadBalancer 'k3d-cosmo-serverlb'   
INFO[0012] Using the k3d-tools node to gather environment information 
INFO[0013] HostIP: using network gateway 172.20.0.1 address 
INFO[0013] Starting cluster 'cosmo'                     
INFO[0013] Starting servers...                          
INFO[0013] Starting Node 'k3d-cosmo-server-0'           
INFO[0019] Starting agents...                           
INFO[0019] Starting Node 'k3d-cosmo-agent-0'            
INFO[0019] Starting Node 'k3d-cosmo-agent-1'            
INFO[0023] Starting helpers...                          
INFO[0023] Starting Node 'cosmo.io'                     
INFO[0024] Starting Node 'k3d-cosmo-serverlb'           
INFO[0030] Injecting records for hostAliases (incl. host.k3d.internal) and for 5 network members into CoreDNS configmap... 
INFO[0035] Cluster 'cosmo' created successfully!        
INFO[0035] You can now use it like this:                
kubectl cluster-info
====== deploy-cert-manager ======
helm repo add jetstack https://charts.jetstack.io
"jetstack" already exists with the same configuration, skipping
helm repo update
Hang tight while we grab the latest from your chart repositories...
...Successfully got an update from the "jetstack" chart repository
Update Complete. ⎈Happy Helming!⎈
helm upgrade --install cert-manager jetstack/cert-manager --namespace cert-manager \
        --create-namespace --version v1.10.0 --set installCRDs=true --wait
Release "cert-manager" does not exist. Installing it now.
E0415 17:33:18.250558 2874933 memcache.go:255] couldn't get resource list for metrics.k8s.io/v1beta1: the server is currently unable to handle the request
E0415 17:33:18.273032 2874933 memcache.go:106] couldn't get resource list for metrics.k8s.io/v1beta1: the server is currently unable to handle the request
NAME: cert-manager
LAST DEPLOYED: Sat Apr 15 17:33:18 2023
NAMESPACE: cert-manager
STATUS: deployed
REVISION: 1
TEST SUITE: None
NOTES:
cert-manager v1.10.0 has been deployed successfully!

In order to begin issuing certificates, you will need to set up a ClusterIssuer
or Issuer resource (for example, by creating a 'letsencrypt-staging' issuer).

More information on the different types of issuers and how to configure them
can be found in our documentation:

https://cert-manager.io/docs/configuration/

For information on how to configure cert-manager to automatically provision
Certificates for Ingress resources, take a look at the `ingress-shim`
documentation:

https://cert-manager.io/docs/usage/ingress/
====== docker-build-traefik-plugins ======
cd ../.. && make docker-build-traefik-plugins VERSION=latest QUICK_BUILD=yes
make[1]: ディレクトリ '/projects/cosmo' に入ります
# cd traefik/plugins/cosmo/authenticate && make vendor 
DOCKER_BUILDKIT=1 docker build . -t traefik-plugins:latest -f dockerfile/traefik-plugins.Dockerfile
[+] Building 1.5s (8/8) FINISHED                                                                                                                                                                           
 => [internal] load build definition from traefik-plugins.Dockerfile                                                                                                                                  0.0s
 => => transferring dockerfile: 404B                                                                                                                                                                  0.0s
 => [internal] load .dockerignore                                                                                                                                                                     0.1s
 => => transferring context: 161B                                                                                                                                                                     0.0s
 => [internal] load metadata for docker.io/library/alpine:3                                                                                                                                           1.4s
 => [stage-0 1/3] FROM docker.io/library/alpine:3@sha256:124c7d2707904eea7431fffe91522a01e5a861a624ee31d03372cc1d138a3126                                                                             0.0s
 => CACHED [stage-0 2/3] RUN apk add --update git                                                                                                                                                     0.0s
 => CACHED [stage-0 3/3] RUN PLUGIN=github.com/wiltonsr/ldapAuth && git clone https://${PLUGIN}.git /plugins-local/src/${PLUGIN} --depth 1                                                            0.0s
 => CACHED [stage-1 2/2] COPY --from=0 /plugins-local /plugins-local                                                                                                                                  0.0s
 => exporting to image                                                                                                                                                                                0.0s
 => => exporting layers                                                                                                                                                                               0.0s
 => => writing image sha256:b3646f8cea7007c645d7b61775e7f38c126740e7b43830cfd8308558c830ac6e                                                                                                          0.0s
 => => naming to docker.io/library/traefik-plugins:latest                                                                                                                                             0.0s
make[1]: ディレクトリ '/projects/cosmo' から出ます
docker tag traefik-plugins:latest localhost:5000/traefik-plugins:latest
docker push localhost:5000/traefik-plugins:latest
The push refers to repository [localhost:5000/traefik-plugins]
0a9c34591fa1: Pushed 
f1417ff83b31: Pushed 
latest: digest: sha256:7664e58a2894ca8c62177d43c0d8da37d94877cb088c4477562b5856d1463c0d size: 738
k3d image import localhost:5000/traefik-plugins:latest -c cosmo
INFO[0000] Importing image(s) into cluster 'cosmo'      
INFO[0000] Starting new tools node...                   
INFO[0000] Starting Node 'k3d-cosmo-tools'              
INFO[0000] Saving 1 image(s) from runtime...            
INFO[0001] Importing images into nodes...               
INFO[0001] Importing images from tarball '/k3d/images/k3d-cosmo-images-20230415173354.tar' into node 'k3d-cosmo-server-0'... 
INFO[0001] Importing images from tarball '/k3d/images/k3d-cosmo-images-20230415173354.tar' into node 'k3d-cosmo-agent-1'... 
INFO[0001] Importing images from tarball '/k3d/images/k3d-cosmo-images-20230415173354.tar' into node 'k3d-cosmo-agent-0'... 
INFO[0003] Removing the tarball(s) from image volume... 
INFO[0004] Removing k3d-tools node...                   
INFO[0004] Successfully imported image(s)               
INFO[0004] Successfully imported 1 image(s) into 1 cluster(s) 
kubectl rollout restart deploy -n kube-system traefik
Error from server (NotFound): deployments.apps "traefik" not found
make: [Makefile:132: docker-build-traefik-plugins] エラー 1 (無視されました)
====== deploy-traefik ======
helm upgrade --install traefik ../../charts/cosmo-traefik -n kube-system \
        --wait \
        -f ./charts/traefik.values.yaml
Release "traefik" does not exist. Installing it now.
Error: An error occurred while checking for chart dependencies. You may need to run `helm dependency build` to fetch missing dependencies: found in Chart.yaml, but missing in charts/ directory: traefik
make: *** [Makefile:83: deploy-traefik] エラー 1
```

--- a/hack/local-run-test/Makefile
+++ b/hack/local-run-test/Makefile
@@ -80,6 +80,7 @@ deploy-cert-manager: create-k3d-cluster helm ## Deploy cert-manager.
 
 deploy-traefik: create-k3d-cluster helm docker-build-traefik-plugins ## Deploy traefik.
 	@echo ====== $@ ======
+	helm dependency update ../../charts/cosmo-traefik 
 	helm upgrade --install traefik ../../charts/cosmo-traefik -n kube-system \
 		--wait \
 	 	-f ./charts/traefik.values.yaml

